### PR TITLE
fix utils loss to make script work and move package to cn.lliiooll.* by lliiooll

### PR DESCRIPTION
**Pull request by NextAlone**
Signed-off-by: NextAlone <12210746+NextAlone@users.noreply.github.com>

# fix utils loss to make script work and move package to cn.lliiooll.*

## 更改内容 / Types of Changes

- 修复 Bugfix
- 增强优化 Enhancement/optimization

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [ ] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Comment by qwq233** [comment]
分俩commit

**Comment by NextAlone** [comment]
> 分俩commit

他应该是一边改一边移动的，这咋弄》

**Comment by qwq233** [comment]
@NextAlone 先把移包的commit 再把fix的commit

**Comment by cinit** [comment]
@qwq233 这样的commit要分开很麻烦吧

**Comment by qwq233** [comment]
@cinit 还好吧 就是因为在移出来的包里才让你先commit移包的

**Review by singleNeuron — DISMISSED** [review]

**Review by qwq233 — CHANGES_REQUESTED** [review]
分完说下w

**Review by qwq233 — APPROVED** [review]

**Review by singleNeuron — DISMISSED** [review]

File: app/src/main/java/cn/lliiooll/QNClient.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        );
+    }
+    
+    /**
+     * 发送一条toast
+     *
+     * @param msg 要显示的消息
+     */
+    public static void toast(String msg) {
+        //Utils.showToast(getApplication(), TOAST_TYPE_INFO, msg, Toasts.LENGTH_LONG);
```
Comment (qwq233) [inline app/src/main/java/cn/lliiooll/QNClient.java:71]: 被注释的代码删下 忘了这垮了

File: app/src/main/java/cn/lliiooll/script/QNScriptFactory.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        script.setEnable(true);
+        initScript(script);
+        enables.put(script.getInfo().getLabel(), script);
+    }
+    
+    /**
+     * 初始化
+     */
+    public static void init() throws FileNotFoundException, EvalError {
+        // File dir = new File("plugins");
```
Comment (qwq233) [inline app/src/main/java/cn/lliiooll/script/QNScriptFactory.java:78]: 同上

File: app/src/main/java/cn/lliiooll/script/QNScriptInfo.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+     *
+     * @param lines 处理的脚本
+     * @return QNScriptInfo实例
+     */
+    public static QNScriptInfo parse(List<String> lines) {
+        QNScriptInfo info = new QNScriptInfo();
+        String line = lines.get(0);
+        if (line.startsWith("//@")) {// 是标识
+            QNScriptInfoLabel infoLabel = QNScriptInfoLabel.parse(line);
+            // System.out.println(infoLabel.toString());
```
Comment (qwq233) [inline app/src/main/java/cn/lliiooll/script/QNScriptInfo.java:52]: 同上

File: app/src/main/java/cn/lliiooll/utils/QNScriptUtils.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    public static List<String> readLines(InputStream is) {
+        List<String> lines = new ArrayList<>();
+        BufferedReader br = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8));
+        String line = "";
+        try {
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            //e.printStackTrace();
```
Comment (qwq233) [inline app/src/main/java/cn/lliiooll/utils/QNScriptUtils.java:37]: 同上

File: app/src/main/java/nil/nadph/qnotified/hook/AbsDelayableHook.java [hunk 5]
```diff
@@ -87,90 +49,93 @@ public static AbsDelayableHook getHookByType(int hookId) {
...
+                SettingEntryHook.get(),
+                DelDetectorHook.get(),
+                PttForwardHook.get(),
+                MuteAtAllAndRedPacket.get(),
+                CardMsgHook.get(),
+                ChatTailHook.get(),
+                FlashPicHook.get(),
+                RepeaterHook.get(),
+                EmoPicHook.get(),
+                //GalleryBgHook.get(),
```
Comment (qwq233) [inline app/src/main/java/nil/nadph/qnotified/hook/AbsDelayableHook.java:61]: 同上

File: app/src/main/java/nil/nadph/qnotified/script/QNScriptManager.java [hunk 6]
```diff
@@ -223,30 +242,37 @@ public static boolean delScript(QNScript script) {
     public static List<QNScript> getScripts() {
         return scripts;
     }
-
+    
     public static void init() {
         if (init) return;
         scriptsPath = getApplication().getFilesDir().getAbsolutePath() + "/qn_script/";
-        for (String code : getScriptCodes()) {
+        /*for (String code : getScriptCodes()) {
```
Comment (qwq233) [inline app/src/main/java/nil/nadph/qnotified/script/QNScriptManager.java:249]: 同上

File: app/src/main/java/cn/lliiooll/QNClient.java (hunk 1 above)
Comment (NextAlone) [reply app/src/main/java/cn/lliiooll/QNClient.java:71]: 好耶

File: app/src/main/java/cn/lliiooll/script/QNScriptFactory.java (hunk 2 above)
Comment (NextAlone) [reply app/src/main/java/cn/lliiooll/script/QNScriptFactory.java:78]: 同上

**Pull request closed without merging**